app/javascript/app/page/page.jsx

```diff
@@ -3,12 +3,15 @@ import Proptypes from 'prop-types';
 
 import TopBar from 'components/topbar';
 import Nav from 'components/nav';
-import styles from './page-styles.scss';
+import Header from 'components/header';
+
+import styles from "./page-styles.scss"; // eslint-disable-line
 
 const Layout = ({ children }) =>
   (<div>
     <TopBar />
     <Nav />
+    <Header />
```

Comment:
This was because you were working with routes and I didn't want to create a page with a route just to display it when it might change. So I put here so you could see it.